apps/frontend/src/views/Results.vue

```diff
@@ -218,7 +218,7 @@ import ExportXCCDFResults from '@/components/global/ExportXCCDFResults.vue';
 import RouteMixin from '@/mixins/RouteMixin';
```

Comment:
the snackbar notification that files aren't loaded in should reflect that it's a sidebar not a hamburger menu anymore

Reply:
Changed icon